benchmarks/DAE/ChemicalAkzoNobel.jmd

```diff
@@ -45,17 +45,16 @@ eqs = [
       D(y₃) ~ r₁ - r₂ + r₃
       D(y₄) ~ -r₂ + r₃ - 2. * r₄
       D(y₅) ~ r₂ - r₃ + r₅
-      D(y₆) ~ ks * y₁ * y₄ - y₆
+      0. ~ ks * y₁ * y₄ - y₆
 ]
 
 ModelingToolkit.@named sys = ModelingToolkit.ODESystem(eqs)
 
 tspan = (0.0, 180.0)
-simpsys = structural_simplify(sys)
-mmprob = ODEProblem(simpsys, [], tspan)
+mmprob = ODEProblem(sys, [], tspan)
 sol = solve(mmprob, Rodas4(),abstol=1/10^14,reltol=1/10^14)
 
-odaeprob = ODAEProblem(simpsys,[],tspan)
+odaeprob = ODAEProblem(structural_simplify(sys),[],tspan)
```

Comment:
accidental commits?

Reply:
it is a merge in SciML:pinn

Reply:
But these commits shouldn't be added to that branch?

Reply:
I just did a rebase to eliminate those.

Reply:
thanks, yes it looks better now